### PR TITLE
chore: update gssmr genesis to v0.9; stub missing ext_ funcs

### PR DESCRIPTION
**Pull request by noot**
## Changes

- draft for now since this causes `ValidateTransaction` and `ApplyExtrinsic` to fail with a codec error

## Tests

```
go test ./lib/runtime/... -short
```

## Issues

- related to #1606

### CODE REVIEW

**Comment by EclesioMeloJunior** [comment]
I'm noticed that in the test function `TestApplyExtrinsic` expects the return of `res, err := babeService.rt.ApplyExtrinsic(ext)` to be `[]byte{0x0, 0x0}` but is receiving `[]byte{0x1, 0x0, 0x6}`, and I found that this errors means:

> ExhaustsResources: Would exhaust the resource of the current block the transaction might be valid

am I missing something while building the extrinsic?

File: lib/babe/build_test.go [hunk 1]
```diff
@@ -83,6 +85,45 @@ func addAuthorshipProof(t *testing.T, babeService *Service, slotNumber, epoch ui
 	babeService.slotToProof[slotNumber] = outAndProof
 }
 
+func createTestExtrinsic(t *testing.T, rt runtime.Instance, genHash common.Hash, nonce uint64) types.Extrinsic {
```
Comment (edwardmack) [inline lib/babe/build_test.go:88]: This func looks the same as func in dot/core/messages_test.go and in lib/runtime/wasmer/exports_test.go, I wonder if we can make one of them public and re-use it?

File: lib/runtime/wasmer/imports.go [hunk 2]
```diff
@@ -1084,6 +1092,13 @@ func ext_default_child_storage_storage_kill_version_1(context unsafe.Pointer, ch
 	storage.DeleteChild(childStorageKey)
 }
 
+//export ext_default_child_storage_storage_kill_version_2
+func ext_default_child_storage_storage_kill_version_2(context unsafe.Pointer, a, b C.int64_t) C.int32_t {
+	logger.Debug("[ext_default_child_storage_storage_kill_version_2] executing...")
+	logger.Warn("[ext_default_child_storage_storage_kill_version_2] unimplemented")
+	return 0
```
Comment (edwardmack) [inline lib/runtime/wasmer/imports.go:1099]: Do we need a TODO, or issue for this since it's unimplemented?

**Review by edwardmack — APPROVED** [review]
Great work, lgtm, just a couple questions.

File: lib/runtime/wasmer/imports.go (hunk 2 above)
Comment (noot) [reply lib/runtime/wasmer/imports.go:1099]: I can open an issue for this

File: lib/babe/build_test.go (hunk 1 above)
Comment (noot) [reply lib/babe/build_test.go:88]: I'd rather not make it public and import one package into the other but I'll move it to utils

Comment (noot) [reply lib/babe/build_test.go:88]: nevermind, that causes a circular import. I'll think about how to do this

**Review by EclesioMeloJunior — APPROVED** [review]
lgtm!

**Pull request merged**